### PR TITLE
coach string and component stubs - round 2

### PR DESCRIPTION
**Pull request by indirectlylit**
### 0.12.x string stubs

* Round 1: #4632
* Round 2: #4642 (this one)
* Round 3: #4651
* Round 4 final: #4662

### Summary

next step after #4632

Stubbing out components and strings for coach reports.

### Reviewer guidance

Check if there are any glaring mistakes, and [help review the strings themselves in this doc](https://docs.google.com/document/d/1OqxXF3rQTAn-2p4SnZo0rsgL8OOlqMcsyK-yeILT6bg/edit#).

### References

* [figma prototype](https://www.figma.com/file/D5wXhmWNGyt60SpGBgFucwZU/Coach-reports-0.12?node-id=1385%3A0)
* [string review doc](https://docs.google.com/document/d/1OqxXF3rQTAn-2p4SnZo0rsgL8OOlqMcsyK-yeILT6bg/edit#)
* [implementation project doc](https://docs.google.com/document/d/1aZ8j5c2q2HkqnPFbLY-bpAAKhZ8wyBF49UaPv0xHtBU/edit#)

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/new/ReportsQuizHeader.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        ];
+      },
+    },
+    methods: {
+      goTo(page) {
+        this.$router.push({ name: 'NEW_COACH_PAGES', params: { page } });
+      },
+    },
+    $trs: {
+      back: 'Back to quizzes',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/new/ReportsQuizHeader.vue]: This is WIP, so I guess the latest iteration with `View all quizzes` is still not in...

Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/new/ReportsQuizHeader.vue]: thanks!

File: kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      },
+      hint() {
+        return this.type === HINT;
+      },
+      error() {
+        return this.type === ERROR;
+      },
+    },
+    $trs: {
+      noAttempt: 'No attempt was made',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue:79]: How about we turn this into active voice: `Learner made no attempts`? Passive often sounds somewhat... imprecise. :thinking:

File: kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      correct: 'Correct answer given',
+      incorrect: 'Incorrect answer given',
+      hint: 'A hint was taken',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue:82]: Same here: `Learner asked for a hint`?

File: kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<script>
+
+  import statusMixin from './status';
+
+  export default {
+    name: 'InProgress',
+    mixins: [statusMixin],
+    $trs: {
+      basicLabelInProgress: '{count, plural, other {In-progress}}',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue:18]: Is the hyphen here really necessary? It's not that without it there's ambiguity.

File: kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      countShortInProgress: '{count, number, integer}',
+      ratioShortInProgress: '{count, number, integer} of {total, number, integer}',
+      countInProgress: '{count, number, integer} {count, plural, other {in-progress}}',
+      ratioInProgress:
+        '{count, number, integer} of {total, number, integer} {count, plural, other {in-progress}}',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue:23]: Same here and below, I see no need for a hyphen.

File: kolibri/plugins/coach/assets/src/views/new/ReportsLessonPage.vue [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    },
+    $trs: {
+      numResources: '{count, number, integer} {count, plural, one {resource} other {resources}}',
+      options: 'Options',
+      editDetails: 'Edit Details',
+      manageResources: 'Manage Resources',
+      back: 'View all lessons',
+      status: 'Status',
+      recipients: 'Recipients',
+      descriptionn: 'Description',
```
Comment (DXCanas) [inline kolibri/plugins/coach/assets/src/views/new/ReportsLessonPage.vue:95]: Typo, but not in string.

File: kolibri/plugins/coach/assets/src/views/new/shared/LessonDetailEditor.vue [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    components: {
+      RecipientSelector,
+      KTextbox,
+      KButton,
+      KRadioButton,
+    },
+    data() {
+      return {
+        lessonTitle: 'Lesson A',
+        lessonDescription: 'Ipsum lorem',
```
Comment (DXCanas) [inline kolibri/plugins/coach/assets/src/views/new/shared/LessonDetailEditor.vue:61]: Flagging.

File: kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue (hunk 2 above)
Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue:79]: addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8

File: kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue (hunk 3 above)
Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/new/shared/Answer.vue:82]: addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8

File: kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue (hunk 4 above)
Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue:18]: addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8

File: kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue (hunk 5 above)
Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue:23]: addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8

**Review by DXCanas — APPROVED** [review]
Sorting out out some misunderstanding one-on-one. No real typos to speak of

**Pull request merged**